update/ansible/playbook/tasks/roles/clickhouse/tasks/main.yml

```diff
@@ -13,7 +15,11 @@
   register: supervisord_socket
 
 - name: Stop and remove clickhouse before update | EL9
-  when: supervisord_socket.stat.exists and (ansible_distribution == 'OracleLinux' or ansible_distribution == 'AlmaLinux') and ansible_distribution_major_version == '9'
+  when:
+    - supervisord_socket.stat.exists
+    - ansible_distribution == 'OracleLinux' or ansible_distribution == 'AlmaLinux'
+    - ansible_distribution_major_version == '9'
+    - not ansible_check_mode
```

Comment:
should we add the same for EL7? How does it fixes upgrade?

Reply:
We can, but it's hardly needed, since we only run this upgrade test in dry-run mode on EL9.